### DISCUSSION TITLE
how about making bool implement IParsable?

### DISCUSSION THREAD

**Discussion by ali50m** [dotnet/runtime#78269] [General] [answered]
Since boolean has owned the TryParse and Parse method for parsing a string to a boolean value.

**Comment by 333fred** [comment]
Moving this to the runtime repository. /cc @tannergooding

**Comment by svick** [comment, marked as answer]
This is a subset of https://github.com/dotnet/runtime/issues/67388 and a duplicate of https://github.com/dotnet/runtime/issues/78523.
